src/main/java/com/streamr/client/protocol/message_layer/MessageID.java

```diff
@@ -7,13 +7,15 @@
     private long timestamp;
     private long sequenceNumber;
     private String publisherId;
+    private String msgChainId;
 
-    public MessageID(String streamId, int streamPartition, long timestamp, long sequenceNumber, String publisherId) {
+    public MessageID(String streamId, int streamPartition, long timestamp, long sequenceNumber, String publisherId, String msgChainId) {
```

Comment:
To follow the pattern, we should have a `MessageIDAdapter` and remove the to/from json code regarding the `MessageID` from `StreamMessageV30Adapter` class.